platform/openide.util.lookup/apichanges.xml

```diff
@@ -53,7 +53,7 @@
             </p>
         </description>
         <class name="ProxyLookup" package="org.openide.util.lookup"/>
-        <issue number="pending"/>
+        <issue number="NETBEANS-2232"/>
```

Comment:
There is no chicken and egg problem, because this should be a JIRA ticket.  AFAIK this will currently get linked to https://issues.apache.org/jira/browse/NETBEANS-2232

Reply:
Fixed in 87a3727e26